doc/api/tls.md

```diff
@@ -1402,6 +1402,12 @@ The `callback` function, if specified, will be added as a listener for the
 
 `tls.connect()` returns a [`tls.TLSSocket`][] object.
 
+**Note:** Unlike the `https` API, `tls.connect()` does not enable the
```

Comment:
We try to move away from 'note' as it doesn't really add much to the docs:
```suggestion
Unlike the `https` API, `tls.connect()` does not enable the
```

Reply:
Thanks!!